src/librustc_unicode/tables.rs

```diff
@@ -145,6 +145,179 @@ pub mod general_category {
         super::trie_lookup_range_table(c, Cc_table)
     }
 
+    pub const L_table: &'static super::BoolTrie = &super::BoolTrie {
```

Comment:
This file is autogenerated and it should not be edited like this, otherwise the edits won't stick. See the reference to `src/etc/unicode.py` at the top of the file